### PR TITLE
AH/release notes migration

### PR DESCRIPTION
**Pull request by AshleyHenry15**
### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

Migrating release notes from our "archived" pages within the repo and also older versions that are within the rstudio.com GH.

**Issue #4282**
**Issue: #11504**

### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

- Create "new" quarto project for "news"
- Migrate all content into a single /docs/news/index.qmd file
- Collaborate with team to decide how we should organize all content
- Organize content after meeting with team
- Have "everyone" review the content to make sure it has been categorized properly
- Make fixes and test
- Then, follow normal review process

**Outstanding!**
- Need to add new release notes link to each guide OR set up a redirect
  - [x]     - Server Admin
  - [ ]     - Desktop - NO (they have a support article currently)
  - [x]     - WB User
  - [x]     - IDE User

PR in docs.posit.co repo to update old release notes links to new URL: https://github.com/rstudio/docs.rstudio.com/pull/1121

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

- Here is the version list from rstudio.com: https://docs.google.com/document/d/1KncCwiUqUfQmR130bm0Q91JLB7BnXrHm39CEQeVnZbE/edit?usp=sharing

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [ ] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [ ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests

### CODE REVIEW

File: docs/news/.gitignore
Comment (melissa-barca) [inline docs/news/.gitignore:25]: I don't think we need a lot of these. I realize you copied this over from a different doc folder, where they're probably also unnecessary, but best not to create more clutter. If this is the only thing holding up merging this PR then I can take care of it in my build PR. 
```suggestion
.DS_Store
kindlegen*
rsconnect
CMake*
!CMakeLists.txt
Makefile
cmake_install.cmake

_site/
_publish.yml
/.quarto/
.bash_history
.cache/*
```

**Review by melissa-barca — APPROVED** [review]

File: docs/news/index.qmd
Comment (MariaSemple) [inline docs/news/index.qmd:62]: Should we leave this section empty until @skyeturriff is running through the release steps and copies over the contents of NEWS.md?
```suggestion
```

File: docs/news/styles.css
Comment (MariaSemple) [inline docs/news/styles.css:1]: Is it intentional that `styles.css` is outside of the `styles` folder?

**Review by MariaSemple — APPROVED** [review]
LGTM! Just a minor nit, probably not a big deal either way

File: docs/news/index.qmd
Comment (melissa-barca) [reply docs/news/index.qmd:62]: After I merge the build changes, I was going to update the [Stable Releases](https://github.com/rstudio/rstudio-pro/wiki/Stable-Releases) documentation with the new release notes process. Either way, it doesn't really matter if this is removed now or not as long as Skye copies over the finalized NEWS.md later.

**Pull request merged**
